packages/eslint-plugin-orbit-components/src/rules/svgUniqueId.ts

```diff
@@ -0,0 +1,88 @@
+import { Rule } from "eslint";
+import { types as t } from "@babel/core";
+
+export default {
+  create: (context: Rule.RuleContext) => {
```

Comment:
I also recommend adding the `meta` object describing the rule's `type` (IMO `"problem"`) and `docs`.